Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -1163,7 +1198,7 @@ def visit_WithStatNode(self, node):
                     if self.check_directive_scope(node.pos, name, 'with statement'):
                         directive_dict[name] = value
         if directive_dict:
-            return self.visit_with_directives(node.body, directive_dict)
+            return self.visit_with_directives(node.body, directive_dict, None)
```

Comment:
A bare `None` argument to a function call (or `True`, or `False`, etc.) is always difficult to interpret.
```suggestion
            return self.visit_with_directives(node.body, directive_dict, contents_directives=None)
```